src/dredd-command.coffee

```diff
@@ -219,9 +219,9 @@ class DreddCommand
       @serverProcess.on 'exit', =>
         logger.info('Backend server process exited')
 
-      @serverProcess.on 'error', (error) =>
-        logger.error('Command to start backend server process failed, exiting Dredd', error)
-        @_processExit(2)
+      @serverProcess.on 'spawnerror', (err) =>
```

Comment:
Reflected in https://github.com/apiaryio/dredd/pull/851/commits/ab90347f3db951441f0cfaa3b30ff9cae901c12d.